transforms/packaging/ray/README.md

```diff
@@ -0,0 +1,37 @@
+# DPK Ray Transforms
+
+## installation
+
+The [transforms](https://github.com/IBM/data-prep-kit/blob/dev/transforms/README.md) are delivered as a standard pyton library available on pypi and can be installed using pip install:
+
+`python -m pip install data-prep-toolkit-transforms-ray`
+
+installing the Ray transforms will also install `data_prep_toolkit_transforms` and `data-prep-toolkit-ray`
+
+## List of Ray Transforms availabe in current package
+
+* code
+	* [code2parquet](https://github.com/IBM/data-prep-kit/blob/dev/transforms/code/code2parquet/ray/README.md)
+	* proglang_select
+	* header_cleanser (Not available on MacOS)
+	* code_quality
+	* repo_level_ordering
```

Comment:
malware

Reply:
malware transforms need some additional work to be included as a pip install and was intentionally excluded from this initial release... It seem to require a docker container and does not fit in the current use case for running these transforms in a notebook.